### PR TITLE
test: download execution tests vectors.

### LINKED ISSUE #98: Download EF test vectors.
We want something similar to what we do in the consensus client, see: https://github.com/lambdaclass/lambda_ethereum_consensus/blob/main/Makefile#L135-L170

For now we care about the blockchain tests, see: https://ethereum.github.io/execution-spec-tests/main/consuming_tests/blockchain_test/. 

We want to filter out tests that are not compatible with Cancun fork as part of this tasks.

Question: should we use `make` or `just` for this?

Acceptance criteria: 
- We should have a command (say a `just` command) that runs all the blockchain tests corresponding to the specified version.

### PR DESCRIPTION
**Pull request by mpaulucci**
**Motivation**
We want to be able to run execution tests from https://github.com/ethereum/execution-spec-tests

**Description**
- Downloads fixtures for EF tests.
- Fixes `sender` for 1559 transactions

Closes #98

### CODE REVIEW

File: ef_tests/tests/tests.rs [hunk 1]
```diff
@@ -30,35 +38,25 @@ fn execute_test(test: TestUnit) {
...
+    use crate::parse_and_execute_test_file;
 
-        for (_k, test) in tests {
-            execute_test(test)
-        }
+    #[test]
+    fn beacon_root_contract_calls_test() {
+        parse_and_execute_test_file("./vectors/cancun/eip4788_beacon_root/beacon_root_contract/beacon_root_contract_calls.json");
     }
 }
```
Comment (MegaRedHand) [inline ef_tests/tests/tests.rs:62]: We could run them all with [datatest_stable](https://docs.rs/datatest-stable/latest/datatest_stable/), but it doesn't support ignoring tests.

**Review by MegaRedHand — APPROVED** [review]
LGTM

File: ef_tests/tests/tests.rs (hunk 1 above)
Comment (mpaulucci) [reply ef_tests/tests/tests.rs:62]: nice, I will investigate for the follow up.

Comment (MegaRedHand) [reply ef_tests/tests/tests.rs:62]: In MLIR we switched to it after experimenting with [libtest-mimic](https://crates.io/crates/libtest-mimic/0.7.3) ([code here](https://github.com/lambdaclass/evm_mlir/blob/fee2671a39a3c7a1089cdd471e0a79cfeb94b537/tests/ef_tests.rs)).

**Pull request merged**